### PR TITLE
Update set-up-orchestrator-explorer-daemonset.md

### PR DESCRIPTION
**Pull request by aquiladayc**
Add instruction regarding collectors option

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md [hunk 1]
```diff
@@ -112,4 +151,4 @@ In some setups, the Process Agent and Cluster Agent cannot automatically detect
   ```
 
   [1]: /containers/cluster_agent/
-  [2]: /containers/cluster_agent/setup/?tab=daemonset#pagetitle
\ No newline at end of file
+  [2]: /containers/cluster_agent/setup/?tab=daemonset#pagetitle
```
Comment (rtrieu) [inline content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md:154]:
```suggestion
  [2]: /containers/cluster_agent/setup/?tab=daemonset
```

Comment (rtrieu) [reply content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md:154]: Not sure why it wants to add #pagetitle to the link, but it looks like it still works without it so suggesting to remove it!

**Review by rtrieu — CHANGES_REQUESTED** [review]
Small change request.

File: content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md (hunk 1 above)
Comment (aquiladayc) [reply content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md]: It's weird, I haven't made any changes on it. Anyway, I'm all good to remove it!

**Review by rtrieu — APPROVED** [review]
LGTM

**Pull request merged**